python/tidy/servo_tidy/tidy.py

```diff
@@ -1052,16 +1052,24 @@ def run_lint_scripts(only_changed_files=False, progress=True, stylo=False):
 
 
 def check_commits(path='.'):
+    """Checks if the test is being run under Travis CI environment"""
+    is_travis = (os.environ.get('TRAVIS') == 'true')
+    number_commits = '-n2' if is_travis else '-n1'
+
     """Gets all commits since the last merge."""
-    args = ['git', 'log', '-n1', '--merges', '--format=%H']
-    last_merge = subprocess.check_output(args, cwd=path).strip()
-    args = ['git', 'log', '{}..HEAD'.format(last_merge), '--format=%s']
+    args = ['git', 'log', number_commits, '--merges', '--format=%H %an']
+    # last_merge stores both the commit hash and the author name of the last merge in the output
+    last_merge_hash, last_merge_author = subprocess.check_output(args, cwd=path).strip().split('\n')[-1].split(None, 1)
```

Comment:
In the case that the author's name is two words, separated by a space, it would not work (it returns `first/middle_name, last_name` instead of `commit_hash, author_name`). Splitting with regards to the `\n` character instead of just `split()` might also suffer from the same issue, as we don't know how many words the author's name might have. So at first, we make sure that we are working with the right string, containing the target commit's hash and author, in the form `commit_hash first_name ... last_name` and then doing a `split(None, 1)` makes sure that the first word in the string is separated from the arbitrary number of words in the author's name.